packages/frontend-shared/src/components/ResultCounts.cy.tsx

```diff
@@ -0,0 +1,28 @@
+import { defaultMessages } from '@cy/i18n'
+import ResultCounts from './ResultCounts.vue'
+
+describe('<ResultCounts />', () => {
+  it('renders expected contents', () => {
+    cy.mount(() => (
+      <ResultCounts
+        totalFailed={1}
```

Comment:
You can also consider adding another test for using a string values like `'3-20'`